api-customs.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"message":"Cannot GET /swagger.json","error":"Not Found","statusCode":404}
```

Comment:
❓

Reply:
This has been broken since we moved to GCP.  I haven't been merging them.